### PR TITLE
Polish the document of 'overview and core concept'

### PR DESCRIPTION
**Pull request by crystaldust**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
None of the above

- Related issues
None

___
### Bug fix
- Bug description.
No bugs

- How to fix?
No fixes
___
### New feature or improvement
- Describe the details and related test reports.
Polish the documents

### CODE REVIEW

**Comment by wu-sheng** [comment]
Why do you disable all the links in the document? I am confusing.

File: docs/en/concepts-and-designs/overview.md [hunk 1]
```diff
@@ -1,45 +1,30 @@
...
-more powerful, SkyWalking could use Service Mesher infra probes to collect data for understanding the whole distributed system.
-In generally, it provides observability capabilities for **service**(s), **service instance**(s), **endpoint**(s).
+SkyWalking provides solutions for observing and monitoring distributed system, in many different scenarios. First of all, like traditional ways, SkyWalking provides auto instrument agents for service, such as Java, C# and NodeJS. At the same time, it provides manual instrument SDKs for Go(Not yet), C++(Not yet). Also with more languages required, risks in manipulating codes at runtime, cloud native infrastructures grow powerful, SkyWalking could use Service Mesher infra probes to collect data for understanding the whole distributed system. In generally, it provides observability capabilities for **service**(s), **service instance**(s), **endpoint**(s).
 
-Service, instance and endpoint concepts are using everywhere in today, so let's be clear what we mean these in SkyWalking first.
+The concept of Service, Instance and Endpoint are used everywhere today, so let's be clear about what they mean in SkyWalking first.
```
Comment (wu-sheng) [inline docs/en/concepts-and-designs/overview.md]: The concepts.

Comment (crystaldust) [reply docs/en/concepts-and-designs/overview.md]: I'll check the format, it should be caused by the markdown editor.

**Comment by crystaldust** [comment]
@wu-sheng I've recovered the lines and make sure the PR only changes the wording

**Pull request merged**